testsuite/tester.nim

```diff
@@ -16,7 +16,7 @@ proc test(t, cmd: string) =
   let nimFile = splitFile(t).name & ".nim"
   if readFile(dir & "tests" / nimFile) != readFile(dir & "results" / nimFile):
     echo "FAILURE: files differ: ", nimFile
-    discard execShellCmd("diff -uNdr " & dir & "results" / nimFile & " " & dir & "tests" / nimFile)
+    discard execShellCmd("diff -uNdw " & dir & "results" / nimFile & " " & dir & "tests" / nimFile)
```

Comment:
What does this do?

Reply:
* `-w` ignores whitespace, needed for an esoteric reason. See comment at https://github.com/nim-lang/c2nim/pull/152/commits/81fe372d7a062e084bb3ee8190c69220bd82326c
* Dropping `-r` ... I think that was because recursive diffing dropped into the `.git/` directory.

I'll rebase this ... Dang it. Too many other changes.

I'll look into this again after [`threadpool.setMaxPoolSize()` is fixed](https://forum.nim-lang.org/t/5061#31781) in Nim. **c2nim** is much lower priority.